revolv/static/css/screen.css

```diff
@@ -5783,8 +5957,74 @@ header.edit-project-header > .container > h1 {
     font-size: 11.57px;
     line-height: 21px;
   }
+
+/* 30-Jun */
```

Comment:
Same thing with this comment. Please update it to describe what the styles below apply to.